service/src/main.rs

```diff
@@ -633,6 +625,22 @@ fn print_events(events: Events, _sender: Sender<String>) {
 					},
 				}
 			},
+			Event::Sidechain(re) => {
+				debug!("{:?}", re);
```

Comment:
Is this still up to date? I believe with the match below you get a `debug` or `trace` message, either way

Reply:
ok i  remove the debug statement.